bedrock/base/templates/includes/site-footer.html

```diff
@@ -1,60 +1,67 @@
-            </ul>
-
-            {% if fx_footer_links %}
-              {% if current_platform %}
-                {{ firefox_footer_links(channel=current_channel, platform=current_platform) }}
-              {% else %}
-                {{ firefox_footer_links(channel=current_channel) }}
-              {% endif %}
-            {% endif %}
-
-            <ul class="links-legal">
-              <li><a href="{{ url('privacy') }}" data-link-type="footer" data-link-name="Privacy">{{ _('Privacy') }}</a></li>
-              <li class="wrap"><a href="{{ url('privacy.notices.websites') }}#cookies" data-link-type="footer" data-link-name="Cookies">{{ _('Cookies') }}</a></li>
-              <li class="wrap"><a href="{{ url('legal.index') }}" data-link-type="footer" data-link-name="Legal">{{ _('Legal') }}</a></li>
-              <li class="clear"><a href="{{ url('legal.fraud-report') }}" data-link-type="footer" data-link-name="Report Trademark Abuse">{{ _('Report Trademark Abuse') }}</a></li>
-            </ul>
-          </div>
-          <div class="col col-3">
-            <ul class="links-social">
+              <li><a href="{{ url('mozorg.contact.contact-landing') }}" data-link-type="footer" data-link-name="Contact Us">{{ _('Contact Us') }}</a></li>
+              <li><a href="{{ donate_url('footer') }}" class="donate" data-link-type="footer" data-link-name="Donate">{{ _('Donate') }}</a></li>
+              <li><a href="https://wiki.mozilla.org/Webdev/GetInvolved/mozilla.org" data-link-type="footer" data-link-name="Contribute to this site">{{ _('Contribute to this site') }}</a></li>
               <li>
-                {{ _('Mozilla:') }}
-                <ul>
+                {{ _('Social:') }}
+                <ul class="social-links">
                   <li><a href="https://twitter.com/mozilla" data-link-type="footer" data-link-name="Twitter (@mozilla)">{{ _('Twitter') }}<span> (@mozilla)</span></a></li>
                   <li><a href="https://www.facebook.com/mozilla" data-link-type="footer" data-link-name="Facebook (Mozilla)">{{ _('Facebook') }}<span> (Mozilla)</span></a></li>
                   <li><a href="https://www.instagram.com/mozillagram/" data-link-type="footer" data-link-name="Instagram (@mozillagram)">{{ _('Instagram') }}<span> (@mozillagram)</span></a></li>
                 </ul>
               </li>
+            </ul>
+          </div>
+          <div class="col col-3">
+            <strong>{{ _('Firefox') }}</strong>
+            <ul class="firefox-links">
+              <li><a href="{{ url('firefox.new') }}" data-link-type="footer" data-link-name="Download Firefox now">{{ _('Download Firefox now') }}</a></li>
+              <li><a href="{{ url('firefox') }}" data-link-type="footer" data-link-name="Firefox Web Browser">{{ _('Firefox Web Browser') }}</li></a>
```

Comment:
This link is a duplicate of the link above, changing only keywords. I'm prepared to accept that these strings are geared towards robots, but creating duplicate links doesn't really do anything for our users. Can I suggest either dropping one of these, or rewording to cover more keywords better in a single instance?